src/pepse/world/daynight/Night.java

```diff
@@ -1,4 +1,39 @@
 package pepse.world.daynight;
 
+import danogl.GameObject;
+import danogl.collisions.GameObjectCollection;
+import danogl.components.CoordinateSpace;
+import danogl.gui.WindowController;
+import danogl.gui.rendering.RectangleRenderable;
+import danogl.gui.rendering.Renderable;
+import danogl.util.Vector2;
+
+import java.awt.*;
+
 public class Night {
+
+    private static Color blackColor = new Color(1);
+    private Vector2 windowDimensions;
+    private static GameObject gameObject;
```

Comment:
no need for this game object  \ the window dimensions. 
where do you use them ? there is no constrctor here !